src/main/java/no/cicero/common/NoBrnValidator.java

```diff
@@ -0,0 +1,58 @@
+package no.cicero.common;
+
+public class NoBrnValidator {
+
+  public static boolean isValid(String brn) {
+    if (brn == null
+      || brn.length() != 9
+      || !brn.matches("^[0-9]+$")) {
+      return false;
+    }
+
+    int checksum = calculateChecksum(brn.substring(0, 8));
+
+    if (checksum == -1
```

Comment:
Is this check redundant, due to only 0-9 being allowed and getNumericValue therefore having only 0-9 as possible return values?